feat(Python) List all integrations (#6412)
getsentry/sentry-docs: src/platforms/python/common/integrations.mdx

---
title: Integrations
description: "The Sentry Python SDK has integrations for popular projects in the Python world."
sidebar_order: 800
redirect_from:
---

> The Sentry SDK for Python supports various projects from the Python world out of the box:

Reviewer: ```suggestion
Sentry's Python SDK supports various projects from the Python world out of the box:
```

- <GuideLink platform="python" guide="quart" />
- <GuideLink platform="python" guide="sanic" />
- <GuideLink platform="python" guide="serverless" />
- <GuideLink platform="python" guide="starlette" />
- Starlite **(Missing docs! Wanna help? [Open a PR](https://github.com/getsentry/sentry-docs))**
- <GuideLink platform="python" guide="tornado" />

## Databases

- [MongoDB](/platforms/python/configuration/integrations/pymongo/)
- PostgreSQL **(Missing docs! Wanna help? [Open a PR](https://github.com/getsentry/sentry-docs))**
- [Redis](/platforms/python/configuration/integrations/redis/)
- Redis Cluster **(Missing docs! Wanna help? [Open a PR](https://github.com/getsentry/sentry-docs))**
- [SQLAlchemy](/platforms/python/configuration/integrations/sqlalchemy/)

## Task Queues

- <GuideLink platform="python" guide="arq" />
- <GuideLink platform="python" guide="celery" />
- huey task queue **(Missing docs! Wanna help? [Open a PR](https://github.com/getsentry/sentry-docs))**
- <GuideLink platform="python" guide="rq" />

## Web Server Interfaces

- [ASGI](/platforms/python/configuration/integrations/asgi/)
- [WSGI](/platforms/python/configuration/integrations/wsgi/)

> ## HTTP clients

Reviewer: ```suggestion
## HTTP Clients
```

- [HTTPX](/platforms/python/configuration/integrations/httpx/)
- requests **(Missing docs! Wanna help? [Open a PR](https://github.com/getsentry/sentry-docs))**

## Cloud Services

- Boto3 **(Missing docs! Wanna help? [Open a PR](https://github.com/getsentry/sentry-docs))**
- [Cloud Resource Context](/platforms/python/configuration/integrations/cloudresourcecontext/)
- [Open Telemenetry](/platforms/python/performance/instrumentation/opentelemetry/)

> ## Data processing

Reviewer: ```suggestion
## Data Processing
```

- <GuideLink platform="python" guide="airflow" />
- <GuideLink platform="python" guide="beam" />
- <GuideLink platform="python" guide="pyspark" />

> ## Function as a service (Faas)

Reviewer: ```suggestion
## Function as a Service (Faas)
```
